deploy/ocm-agent-operator-managednotifications/10-managednotifications-pdb.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: ocmagent.managed.openshift.io/v1alpha1
+kind: ManagedNotification
+metadata:
+  name: sre-pdb-managed-notifications
+  namespace: openshift-ocm-agent-operator
+spec:
+  notifications:
+    - activeBody: |-
+        The pod disruption budget is below the minimum disruptions allowed level and is not satisfied. The number of current healthy pods is less than the desired healthy pods. For more information, please refer to https://github.com/openshift/runbooks/blob/master/alerts/cluster-kube-controller-manager-operator/PodDisruptionBudgetLimit.md
+      name: PodDisruptionBudgetLimit
+      resendWait: 24
```

Comment:
Will this be spammy for customers?

Reply:
i am happy to change to any other number you think is right

Reply:
I'm asking for an assessment and opinion.